### PR TITLE
LEAF-4199 - Text updates

### PR DESCRIPTION
**Pull request by mgaoVA**
3 changes:
1. When there's a disabled account associated with a "person designated" request, the workflow shows a visible "\*NEEDS REASSIGNMENT\*" message.
2. Text related to "restore records" has been re-worded to clarify its status.
3. In the Inbox, approverName can contain the label of a field in the event there's a disabled account in a "person designated" field, which can contain HTML formatting. This change scrubs HTML from the text to ensure formatting is consistent.

## Potential Impact
- Admins may require notification to explain the "NEEDS REASSIGNMENT" message

## Testing
1. Set up a record that's pending a "person desginated" field
2. Disable the account associated with the "person desginated" field
- [ ] The workflow for the record should show "NEEDS REASSIGNMENT"
- [ ] The Inbox (Organize by Forms + View as Admin) should show "NEEDS REASSIGNMENT" as one of the sections

### CODE REVIEW

File: LEAF_Request_Portal/templates/print_form.tpl [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!--{if $deleted > 0}-->
-    <div style="font-size: 36px"><img src="dynicons/?img=emblem-unreadable.svg&amp;w=96" alt="Unreadable"
-            style="float: left" /> Notice: This request has been marked as deleted.<br />
+    <div style="font-size: 36px"><img src="dynicons/?img=emblem-unreadable.svg&amp;w=96"
```
Comment (aerinkayne) [inline LEAF_Request_Portal/templates/print_form.tpl:2]: These alt attributes should just be empty strings (rather than absent)

Comment (raphaelfretter) [reply LEAF_Request_Portal/templates/print_form.tpl:2]: Why does it need an alt tag? Screen readers will just ignore the element if there isn't one.

Comment (mgaoVA) [reply LEAF_Request_Portal/templates/print_form.tpl]: Thanks, updated.

File: LEAF_Request_Portal/templates/reports/LEAF_Inbox.tpl [hunk 2]
```diff
@@ -189,6 +189,12 @@
         },
     };
 
+    function scrubHTML(input) {
```
Comment (Pelentan) [inline LEAF_Request_Portal/templates/reports/LEAF_Inbox.tpl:192]: A concern I have with this.  It doesn't do what it says it's doing.  It's just wrapping it in a div.  I would recommend either a different name or fully explain what it's _meant_ to do in comments inside the function itself.

File: LEAF_Request_Portal/sources/FormWorkflow.php [hunk 3]
```diff
@@ -206,8 +206,8 @@ private function includePersonDesignatedData(array $srcRecords, array $pdRecords
                     $approver = $dir->lookupEmpUID($dRecords[$v['recordID']]['data']);
                     
                     if (empty($approver[0]['Fname']) && empty($approver[0]['Lname'])) {
-                        $srcRecords[$i]['description'] = $srcRecords[$i]['stepTitle'] . ' (' . $dRecords[$v['recordID']]['name'] . ')';
-                        $srcRecords[$i]['approverName'] = $dRecords[$v['recordID']]['name'];
+                        $srcRecords[$i]['description'] = $srcRecords[$i]['stepTitle'] . ' ( *NEEDS REASSIGNMENT* ' . $dRecords[$v['recordID']]['name'] . ')';
```
Comment (Pelentan) [inline LEAF_Request_Portal/sources/FormWorkflow.php:209]: Thinking 508 compliance.  This _looks_ like it's trying to flag something for a viewer's attention.  Only what if they can't "view"?  Do we want to create a tag system for a "reader" to catch and give audible alert?

**Review by Pelentan — CHANGES_REQUESTED** [review]

File: LEAF_Request_Portal/sources/FormWorkflow.php (hunk 3 above)
Comment (mgaoVA) [reply LEAF_Request_Portal/sources/FormWorkflow.php:209]: A screen reader would simply read out the text.

This prepends the text to the step's title when there's no first/last name associated with the approver, so it'll be seen/heard anywhere the stepTitle shows up.

**Review by jampaul3 — APPROVED** [review]

**Review by raphaelfretter — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]

**Review by shaneodd — APPROVED** [review]

**Comment by shaneodd** [comment]
I did not see any issues arising from these changes on local testing.

**Pull request merged**